test/libraries/Safe150Migration.spec.ts

```diff
@@ -0,0 +1,771 @@
+import { expect } from "chai";
+import hre, { ethers, deployments } from "hardhat";
+import { AddressZero } from "@ethersproject/constants";
+import { getSafeWithSingleton, migrationContractTo150, getSafeSingletonAt, getMock } from "../utils/setup";
+import deploymentData from "../json/safeDeployment.json";
+import safeRuntimeBytecode from "../json/safeRuntimeBytecode.json";
```

Comment:
We use a separate file for the bytecode (instead of the build artefacts) to make the tests work also for future versions?

Reply:
Hmm, I was torn because there are no guarantees that the next version will be compatible with the upgrade contract